nutanix/resource_nutanix_virtual_machine_test.go

```diff
@@ -257,7 +257,7 @@ func TestAccNutanixVirtualMachine_PowerStateMechanism(t *testing.T) {
 				ResourceName:            resourceName,
 				ImportState:             true,
 				ImportStateVerify:       true,
-				ImportStateVerifyIgnore: []string{"disk_list"},
+				ImportStateVerifyIgnore: []string{"power_state_mechanism_config", "power_state_mechanism"},
```

Comment:
?Can't we add those values in the read function?

Reply:
yes let change and check them

Reply:
they are in the read function but they depend on a config to be set and they have the same attribute but with different structure, the `power_state_mechanism` will be deprecated, so I put them in ImportStateVerifyIgnore array due to a state error